controller/cache/cache.go

```diff
@@ -114,6 +120,7 @@ func init() {
 	clusterCacheListSemaphoreSize = env.ParseInt64FromEnv(EnvClusterCacheListSemaphore, clusterCacheListSemaphoreSize, 0, math.MaxInt64)
 	clusterCacheAttemptLimit = int32(env.ParseNumFromEnv(EnvClusterCacheAttemptLimit, int(clusterCacheAttemptLimit), 1, math.MaxInt32))
 	clusterCacheRetryUseBackoff = env.ParseBoolFromEnv(EnvClusterCacheRetryUseBackoff, false)
+	clusterCacheBatchEventsProcessing = env.ParseBoolFromEnv(EnvClusterCacheBatchEventsProcessing, false)
```

Comment:
Two broader questions:

1. Should we expose this as part of the `argocd-cmd-params` [config map](https://argo-cd.readthedocs.io/en/stable/operator-manual/argocd-cmd-params-cm-yaml/)?
2. Should we default it to `true`?

Reply:
Thank you for review @rumstead 

> 1. Should we expose this as part of the argocd-cmd-params [config map](https://argo-cd.readthedocs.io/en/stable/operator-manual/argocd-cmd-params-cm-yaml/)?

`ARGOCD_CLUSTER_CACHE_BATCH_EVENTS_PROCESSING` option was exposed as env variable the same way as all other `clustercache` (part of [gitops-engine](https://github.com/argoproj/gitops-engine/blob/66d645e5481fe3d7400b93b5b9abdb59c7f1fdf0/pkg/cache/cluster.go#L163)) options are exposed.

> 2. Should we default it to true?

Currently, the default value should be `false`; that doesn't change the event's processing mode, so no one can be affected by this change. To change the event's processing mode, you must explicitly set the `ARGOCD_CLUSTER_CACHE_BATCH_EVENTS_PROCESSING` env variable to `true`.